src/ComposerRequireChecker/FileLocator/LocateComposerPackageSourceFiles.php

```diff
@@ -55,9 +55,9 @@ private function locateFilesInFilesInFilesDefinitions(array $locations) : Genera
         foreach ($locations as $location) {
             if (is_file($location)) {
                 yield $location;
+            } elseif (is_dir($location)) {
+                yield from $this->extractFilesFromDirectory($location);
             }
-
-            yield from $this->extractFilesFromDirectory($location);
```

Comment:
Now that I think of it... it may return `false` for both dirs and files if the symlink is broken.